torchgeo/datasets/etci2021.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""ETCI 2021 dataset."""
+
+import glob
+import os
+import shutil
+from typing import Callable, Dict, List, Optional
+
+import numpy as np
+import torch
+from PIL import Image
+from torch import Tensor
+
+from .geo import VisionDataset
+from .utils import download_and_extract_archive
+
+
+class ETCI2021(VisionDataset):
+    """ETCI 2021 Flood Detection dataset.
+
+    The `ETCI2021 <https://nasa-impact.github.io/etci2021/>`_
+    dataset is a dataset for flood detection
+
+    Dataset features:
+    * 33,405 VV & VH Sentinel-1 Synthetic Aperture Radar (SAR) images
+    * 2 binary masks per image representing water body & flood, respectively
+    * 2 polarization band images (VV, VH) of 3 RGB channels per band
+    * 3 RGB channels per band generated by the Hybrid Pluggable
+    Processing Pipeline 'hyp3'
+    * Images with 5x20m per pixel resolution (256x256) px) taken in
+    Interferometric Wide Swath acquisition mode
+    * Flood events from 5 different regions
+
+    Dataset format:
+    * VV band three-channel png
+    * VH band three-channel png
+    * water body mask single-channel png where no water body = 0, water body = 255
+    * flood mask single-channel png where no flood = 0, flood = 255
+
+    Dataset classes:
+    1. no flood/water
+    2. flood/water
+
+    If you use this dataset in your research, please add the following to your
+    acknowledgements section:
+
+    'The authors would like to thank the NASA Earth Science Data Systems Program,
+    NASA Digital Transformation AI/ML thrust, and IEEE GRSS for organizing
+    the ETCI competition'.
```

Comment:
```suggestion
        The authors would like to thank the NASA Earth Science Data Systems Program,
        NASA Digital Transformation AI/ML thrust, and IEEE GRSS for organizing
        the ETCI competition'.
```
Quote blocks can be created by indenting: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks

Reply:
Fixed per suggestion